precompiles/utils/xcm-codec/src/tests.rs

```diff
@@ -733,45 +733,42 @@ fn read_complex_solidity_function() {
 #[test]
 fn junctions_decoder_works() {
 	let writer_output = Writer::new()
-		.write(Junctions::X1(Junction::OnlyChild))
+		.write(XCMJunctions(Junctions::X1(Junction::OnlyChild)))
```

Comment:
Small grumble: Following Rust naming conventions it should be `XcmJunctions`.

Reply:
done, thanks! 👍